sdk/python/jobs/automl-standalone-jobs/automl-image-classification-multiclass-task-fridge-items/automl-image-classification-multiclass-task-fridge-items.ipynb

```diff
@@ -640,6 +647,13 @@
     "    compute=compute_name,\n",
     "    # name=\"dpv2-image-classification-job-02\",\n",
     "    experiment_name=exp_name,\n",
+    "    properties={\n",
```

Comment:
@KonakanchiSwathi - Please revert the changes used for testing purroses.

Reply:
removed